docs/rules/no-unused-expressions.md

```diff
@@ -34,6 +41,34 @@ f(0), {}
 a && b()
 ```
 
+If you enable the `allowShortCircuit` and/or `allowTernary` options when required you can use expressions like the following but keep in mind that the same restriction on expression side effects still applies:
+
+```js
+a && b()
+```
```

Comment:
@IanVS Thanks for the clarification and for pointing me at that example. I'll take some time today and merge the code blocks and get the docs to match that pattern.